### PR TITLE
Add output chunking for Cromwell provisioner

### PR DESCRIPTION
**Pull request by apmasell**
Change the Cromwell output provisioner to create more hierarchical directory
structures to avoid overloading a single directory.

### CODE REVIEW

File: vidarr-cromwell/README.md [hunk 1]
```diff
@@ -34,6 +35,16 @@ with two arguments, the file to copy (which will be provided as the argument
 collect the permanent location for the file (`"storagePathField"`), the MD5
 (`"md5Field"), and the file size ("fileSizeField").
 
+Your file system probably will not appreciate having thousands of files dumped
+in a single output directory, so the `"chunks"` parameter will create a
+heirarchy of directories based on the workflow run identifier. The numbers
```
Comment (iainrb) [inline vidarr-cromwell/README.md:40]: heirarchy -> hierarchy

File: vidarr-cromwell/README.md [hunk 2]
```diff
@@ -34,6 +35,16 @@ with two arguments, the file to copy (which will be provided as the argument
...
+determine the number of characters to use in each directory. For example, `[2,
+4]` will take an ID of the form `AABBBBCCCCCCCCCCCCCC` and produce an output
+path that is `AA/BBBB/AABBBBCCCCCCCCCCCCCC`. Once files have been provisioned
+out, it is possible to change the chunking scheem, but the exist files are
```
Comment (iainrb) [inline vidarr-cromwell/README.md:44]: scheem -> scheme
exist -> existing

**Review by iainrb — APPROVED** [review]
A few typos in the README, otherwise looks good

File: vidarr-cromwell/README.md (hunk 1 above)
Comment (apmasell) [reply vidarr-cromwell/README.md]: I can't spell this word to save my life and it comes up a lot in computing.

**Review by pruzanov — APPROVED** [review]

**Pull request merged**